index.js

```diff
@@ -1,11 +1,29 @@
-function scuberGreetingForFeet(){
+function scuberGreetingForFeet(feet){
   // Write your code here!
+  if(feet <= 400){
+    return 'This one is on me!'
+  } else if(feet >400 && feet < 2000){
+    return 'That will be twenty bucks.';
+  } else if(feet > 2000 && feet <= 2500){
+    return 'I will gladly take your thirty bucks.'
+  } else if(feet > 2500){
```

Comment:
Same idea here. 
```suggestion
  } else {
```

Reply:
I see. I guess I'm not that comfortable yet with the flow so I'm being redundant.